src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -38,7 +34,8 @@
   </Target>
   <!-- Copy over the performance runners to the test directory-->
   <Target Name="CopyXunitExecutionFiles" AfterTargets="CopyTestToTestDirectory" Condition="'$(Performance)' == 'true'">
-
+    <Error Condition="'$(XunitPerfRunnerPackageVersion)' == ''" Text="Xunit performance package version was not specified" />
+    <Error Condition="Exists('$(XunitPerfAnalysisPath)')" Text="Xunit performance package was not found" />
```

Comment:
Did you test this? If you had I would expect you to hit this error as I believe the check should be !Exists